contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -145,7 +145,7 @@ <h2 id="contribute">Contribution Details</h2>
                             <input class="form-control" id="contribution_Admins"
                                    name="contribution_contributionAdmins"
                                    placeholder="Give a list of admin names, separated by comma"
-                                   type="text" value="{{ post.contributionAdmins|join(' ') }}"></div>
+                                   type="text" value="{{ post.contributionAdmins|join(',') }}"></div>
```

Comment:
Could we add a space after the comma for both displaying and editing? Otherwise, it looks good to me. I assume, since you are using the strip function, it would remove the spaces when processing in the Python code. Thanks.

Reply:
updated.

Reply:
Thanks.